types/test.ts

```diff
@@ -0,0 +1,79 @@
+import * as keys from 'ssb-keys'

```

Comment:
I'd much prefer that all tests where in the test directory, is it possible to move this?

Reply:
I was trying to contain all the typing related in an extra folder, to not spill extra files all over the place. Will have a look if this is possible.

Reply:
Looking at the code[1] this is not possible. dtslint expects the test file along the type definition.


[1] https://github.com/Microsoft/dtslint/blob/master/src/index.ts#L46 && https://github.com/Microsoft/dtslint/blob/master/src/index.ts#L61 && https://github.com/Microsoft/dtslint/blob/master/src/index.ts#L85